htdocs/product/card.php

```diff
@@ -2406,7 +2406,7 @@
 				}
 			}
 
-			if (empty($conf->global->PRODUCT_DISABLE_ACCOUNTING)) {
+			if (empty($conf->global->PRODUCT_DISABLE_ACCOUNTING) && isModEnabled('accounting')) {
```

Comment:
A lot of user does not use the accounting module.
But they need to store the accounting account on product (for information or for export by other accounting module like the ACCOUNTING (SIMPLIFIED) module.
This is the reason of the option PRODUCT_DISABLE_ACCOUNTING. If you really don't need it and don't want to keep the field empty, you can enable it to remove the field completely.